src/main.ts

```diff
@@ -0,0 +1,20 @@
+import * as core from "@actions/core";
```

Comment:
do we really need to introduce ts for a single file that doesn't benefit from using ts in the first place?

Reply:
I only used TS because it was in the requirements, I think it can be removed.

Reply:
yes please, sorry for the confusion